packages/fs-detectors/test/unit.framework-detector.test.ts

```diff
@@ -164,11 +164,23 @@ describe('DetectorFilesystem', () => {
     const files = {};
     const fs = new VirtualFilesystem(files);
 
-    fs.writeFile('file.txt', 'Hello World');
+    fs.writeFile('file.txt', true, 'Hello World');
```

Comment:
Lets use an object instead of multiple params.

Also, can we follow the Node.js `fs.writeFile` API here (not sure if there is an equivalent)

Reply:
Sounds good!

I initially followed the nodejs `writeFile` API here, but will add an `options` object as the third parameter 🙌 

<img width="510" alt="image" src="https://user-images.githubusercontent.com/1278714/187927390-c3db2eda-4c0d-40f2-b510-b16a761eee10.png">